src/hooks/useLoadAccountData.js

```diff
@@ -15,6 +17,9 @@ import { uniswapLoadState } from '../redux/uniswap';
 import { walletConnectLoadState } from '../redux/walletconnect';
 import { logger, promiseUtils } from '../utils';
 
+const loadTopMoversToCache = () =>
+  queryCache.prefetchQuery('topMoversFromStorage', getTopMovers);
```

Comment:
With long strings like this I'm very likely to have typos.  If we have more queries in the future we should consider making it a constant (like we did with the routes)